### PR TITLE
adding designs and hacks

### PR DESCRIPTION
**Pull request by bnvk**

### CODE REVIEW

File: 2015-04-25-designs-and-hacks.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Designs and Hacks, Apr 25, 2015
+
+Lots of things that we've talking about for the last couple months. Time to push some pixels and code and breath life into these objective. Let’s meet and work on driving the Open Soure Design movement forward. Everyone who wants to contribute is welcome :)
+
+
+## Date & Location
+
+- **Date:** Sun, 25 April 2015
```
Comment (jancborchardt) [inline 2015-04-25-designs-and-hacks.md]: The 25th is Saturday ;)

File: 2015-04-25-designs-and-hacks.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- **Time:** 14:00 – 18:00 CET (UTC +1:00)
+- **Where:** @bnvk Haus
+	- Gneisenaustraße 52
+	- Kreuzberg, Berlin
+	- Near U7-Südstern stop
+- **Where (remote):** Participate via [video chat](https://appear.in/osd-designs-and-hacks)
```
Comment (jancborchardt) [inline 2015-04-25-designs-and-hacks.md]: I would say we just use only https://appear.in/opensourcedesign for any and all of our video calls. Then we just have to open it from browser history and it makes it much easier to invite people.

**Comment by jancborchardt** [comment]
@bnvk great on putting that up! Added some notes. See you remotely!

@t0keshu cool that you are in as well! I think I know you via Hugo Roy ;)

**Comment by jancborchardt** [comment]
@bnvk great on putting that up! Added some notes. See you remotely!

@tOkeshu cool that you are in as well! I think I know you via @hugoroy, could that be?

**Comment by bnvk** [comment]
@jancborchardt good catches fixes!

**Comment by jancborchardt** [comment]
Nice! See you on Saturday!

**Pull request merged**